fsharp/falco/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  website: github.com/pimbrouwers/Falco
+  version: 1.0.10-alpha
```

Comment:
I prefer to use **stable** release, no `alpha`

@pimbrouwers I found only `alpha` on https://www.nuget.org/packages/Falco

Reply:
You forget to update this. Moreover, I prefer having `X.Y` format -> **1.1**